linera-views/src/key_value_store_view.rs

```diff
@@ -424,6 +428,9 @@ where
         let mut missed_indices = Vec::new();
         let mut vector_query = Vec::new();
         for (i, index) in indices.into_iter().enumerate() {
+            if index.len() > COMMON_MAX_KEY_SIZE {
```

Comment:
I don't think this needs to be a static constant in this particular file.

Reply:
Ok, I changed it accordingly. Sorry for not having understood right away.